api/room/models.py

```diff
@@ -36,12 +36,19 @@ class Room(Base, Utility):
         ForeignKey('locations.id', ondelete="CASCADE"),
         nullable=True
     )
+    structure_id = Column(
+        Integer,
+        ForeignKey('structure.id', ondelete="CASCADE"),
+        nullable=True   # TODO: Change this to False
```

Comment:
Great work @RachaelNantale, but should `nullable` not be `False`?

Reply:
Thanks, @ThaDeveloper,  as advised by @joshuaocero,  I used `True` because of the already existing data in the database would not be lost if my migration is run. As explained in the `TODO` comment.  This will be changed to False after the migration from v1 to v2.